jvm/src/test/scala/eval/ParallelEvaluationSpec.scala

```diff
@@ -0,0 +1,342 @@
+package com.azavea.maml.eval
```

Comment:
This file looks long because it includes all of the `NaiveInterpreter` tests, but with an `unsafeRunSync` attached, to make sure that they interpret expressions the same